### PR TITLE
added ability to use template cookbook for login.defs

### PR DESCRIPTION
**Pull request by ekelson-bcove**

### CODE REVIEW

**Comment by ekelson-bcove** [comment]
I noticed the login.defs file has a lot of comments and such.  A custom template would allow an org to achieve more customization.

File: metadata.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@
 license          'Apache-2.0'
 description      'Installs and configures operating system hardening'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '3.1.0'
```
Comment (artem-sidorenko) [inline metadata.rb]: @ekelson-bcove please do not bump the cookbook version. This is done by maintainers before the release is done

**Comment by ekelson-bcove** [comment]
Of course. How do you want me to remove? Another pull request or want to
just do that upstream.

**Comment by ekelson-bcove** [comment]
Fixed above

**Comment by artem-sidorenko** [comment]
@ekelson-bcove thank you!

**Pull request merged**